notebooks/gnn/mag240m_pg.ipynb

```diff
@@ -0,0 +1,405 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load the MAG240M Dataset into a Property Graph\n",
+    "\n",
+    "The notebook is meant to stress test loading data into a property graph. \n",
+    "The notebook requires that the data has already been downloaded \n",
+    "\n",
+    "\n",
+    "    from ogb.lsc import MAG240MDataset\n",
+    "    dataset = MAG240MDataset(root = base_dir)\n",
+    "    dataset.download()\n",
+    "\n",
+    "\n",
+    "num_gpus = Enter the number of GPUs that you have in the cluster.  This will determine the number of records loaded\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Feature variable\n",
+    "skip_edges = False\n",
+    "skip_features = True\n",
+    "\n",
+    "load_paper_features = True\n",
+    "load_paper_labels = True\n",
+    "load_paper_year = True\n",
+    "\n",
+    "load_affiliation_edges = False\n",
+    "load_writes_edges = True\n",
+    "load_cites_edges = False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import needed libraries. \n",
+    "# We recommend using the [cugraph_dev](https://github.com/rapidsai/cugraph/tree/branch-22.10/conda/environments) env through conda\n",
+    "\n",
+    "from dask.distributed import Client, wait\n",
+    "from dask_cuda import LocalCUDACluster\n",
+    "from cugraph.dask.comms import comms as Comms\n",
+    "import cugraph.dask as dask_cugraph\n",
+    "import cugraph\n",
+    "import dask_cudf\n",
+    "import cudf\n",
+    "import time\n",
+    "import numpy as np\n",
+    "import math"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# size of dataset\n",
+    "class Dataset:\n",
+    "    def __init__(self):\n",
+    "        self.num_papers  = 121751666\n",
+    "        self.num_authors = 122383112\n",
+    "        self.num_institutions = 25721\n",
+    "        self.num_paper_features = 768\n",
+    "        self.num_classes = 153\n",
+    "        self.num_cite_edges = 1297748926\n",
+    "        self.num_write_edges = 386022720\n",
+    "        self.num_affiliated_edges = 44592586\n",
+    "    \n",
+    "    def adjust_by(self, factor):\n",
+    "        self.num_papers = math.floor(self.num_papers * factor)\n",
```

Comment:
In python 3, I don't think there's a different between `math.floor()` and `int()` for positive numbers (I'm assuming none of these should be negative?).  If you go with `int()`, you can remove `import math` above.

Reply:
you are correct that in Python 3 the end results is similar (assuming non-negative).   I used floor since I wanted to explicitly state that I wanted to round-down.   
I'll shift to int since it does save an import